test/sockend.js

```diff
@@ -234,3 +234,47 @@ test.cb(`Sockend ns late bound req&res`, (t) => {
         const client = ioClient(`http://0.0.0.0:${port}/${namespace}`);
     });
 });
+
+test.cb('Sockend req&res works without callback', (t) => {
```

Comment:
can we actually demonstrate it with a simple requester / responder pair instead of a sockend?

Reply:
It was my initial idea but thought using original methods and test the case with the discovered way.

`requester.sock.send` would have to be used over `requester.send`, because `send` defines a callback to be able to return a promise when the callback is not defined in the call. (see: https://github.com/dashersw/cote/blob/43c84bb/src/components/requester.js#L101-L108)

But obviously that'd be faster and covering the case we have here.

What do you think?